packages/manager/modules/core/src/config/config.provider.js

```diff
@@ -53,10 +53,15 @@ export default class CoreConfig {
   }
 
   getApplicationBaseURL(id) {
-    const applicationURL = this.environment.getApplicationURL(id);
-    if (applicationURL === this.getCurrentApplicationURL()) {
+    const applicationURL = this.environment.getApplications()[id]?.publicURL;
+    const currentAppPublicURL = this.environment.getApplications()[
```

Comment:
Is `getCurrentApplicationURL` used elsewhere ? Should we update `getCurrentApplicationURL` method ?

Reply:
It seems that it's not used elsewhere. I have updated this method in order to get the publicURL of current application.